optuna/cli.py

```diff
@@ -656,14 +680,17 @@ def take_action(self, parsed_args: Namespace) -> int:
 
 
 class _StorageUpgrade(_BaseCommand):
-    """Upgrade the schema of a storage."""
+    """Upgrade the schema of an RDB storage."""
 
     def take_action(self, parsed_args: Namespace) -> int:
         storage_url = _check_storage_url(parsed_args.storage)
-        if storage_url.startswith("redis"):
-            self.logger.info("This storage does not support upgrade yet.")
+        try:
+            storage = RDBStorage(
+                storage_url, skip_compatibility_check=True, skip_table_creation=True
+            )
+        except sqlalchemy.exc.ArgumentError:
+            self.logger.info("Invalid RDBStorage URL.")
```

Comment:
It might be an error message, but the original implementation uses `self.logger.info`. Just a question, but do we have any reason not to use `self.logger.error` here?

Reply:
Thank you for your comment.

The reason I used `logger.info(...)` here is because the previous implementation used `logger.info(...)` when given a Redis storage url, so I followed that. However, as you pointed out, using `logger.error(...)` sounds reasonable.

```suggestion
            self.logger.error("Invalid RDBStorage URL.")
```